src/torchcodec/transforms/_decoder_transforms.py

```diff
@@ -0,0 +1,85 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from typing import Sequence
+
+from torch import nn
+
+
+@dataclass
+class DecoderTransform(ABC):
+    """Base class for all decoder transforms.
+
+    A *decoder transform* is a transform that is applied by the decoder before
+    returning the decoded frame.  Applying decoder transforms to frames
+    should be both faster and more memory efficient than receiving normally
+    decoded frames and applying the same kind of transform.
+
+    Most `DecoderTransform` objects have a complementary transform in TorchVision,
```

Comment:
Annoyingly, single backticks in `rst` means italics. I think you wanted those to be code, like in markdown? For that we need double backticks (there are other instances of single backticks below and maybe in other files?)


```suggestion
    Most ``DecoderTransform`` objects have a complementary transform in TorchVision,
```